components/donateform.js

```diff
@@ -39,7 +39,12 @@ class DonateForm extends Component {
       return
     }
     try {
-      const { frequency, firstName, lastName, email, amount } = formValues
+      let { frequency, firstName, lastName, email, amount } = formValues
+      let description = 'Donation'
+      if (frequency === '2022_fall_event') {
```

Comment:
Thoughts on making `2022_fall_event` a reusable constant? I'm seeing it in a couple of places.

Reply:
great idea kev!!!